storage/operation/transaction_results.go

```diff
@@ -0,0 +1,166 @@
+package operation
```

Comment:
this file is identical to the same file in storage/badger/operation/

Except the following functions are removed:
- BatchInsertTransactionResult
- BatchIndexTransactionResult